streamer/bitrate_configuration.py

```diff
@@ -114,7 +120,7 @@ def get_output_format(self) -> str:
     # TODO(#31): add support for configurable output format per-codec
     if self.get_base_codec() == VideoCodec.VP9:
       return 'webm'
-    elif self.get_base_codec() == VideoCodec.H264:
+    elif self.get_base_codec() == VideoCodec.H264 or VideoCodec.HEVC:
```

Comment:
This doesn't do what you want.  You need to do `self.get_base_codec() == VideoCodec.HEVC`; or you can replace it with `self.get_base_codec() in {VideoCodec.H264, VideoCodec.HEVC}`.